exporters/trace/jaeger/jaeger.go

```diff
@@ -214,14 +214,26 @@ func (e *Exporter) ExportSpans(ctx context.Context, ss []*export.SpanSnapshot) e
 		return nil
 	}
 
-	for _, span := range ss {
-		// TODO(jbd): Handle oversized bundlers.
-		err := e.bundler.Add(span, 1)
-		if err != nil {
-			return fmt.Errorf("failed to bundle %q: %w", span.Name, err)
+	var bundlerErr error
+	done := make(chan struct{})
+	go func() {
+		for _, span := range ss {
+			err := e.bundler.AddWait(ctx, span, 1)
```

Comment:
We need to perserve this TODO as it is not addressed here:

```suggestion
			// TODO(jbd): Handle oversized bundlers.
			err := e.bundler.AddWait(ctx, span, 1)
```